README.md

```diff
@@ -24,104 +18,67 @@ of active locales is:
 
-3. Next, create an environment variable called `CONTENT_TRANSLATED_ROOT`
-   containing the path to the *translated-content* repo’s `files` directory. You
-   could do this for a single session like so:
+3. Next, create an environment variable called `CONTENT_TRANSLATED_ROOT` containing the path to the *translated-content* repo’s `files` directory. You could do this for a single session like so:
 
    ```bash
    export CONTENT_TRANSLATED_ROOT=/path/to/translated-content/files
    ```
 
-   But you'll have to newly-set this every time you open up a new terminal
-   window. Instead, you could put the environment variable setting in an `.env`
-   file in the root of your content repo. This is most easily done using the
-   following command:
+   But you'll have to newly-set this every time you open up a new terminal window. Instead, you could put the environment variable setting in an `.env` file in the root of your content repo. This is most easily done using the following command:
 
    ```bash
    echo CONTENT_TRANSLATED_ROOT=/path/to/translated-content/files >> .env
    ```
 
    (the `.env` file will be created for you if it does not already exist.)
 
-4. Now you've got this set up, enter the command `yarn start` to begin the local
-   testing server running at `localhost:5042`.
+4. Now you've got this set up, enter the command `yarn start` to begin the local testing server running at `localhost:5042`.
 
 ### Working in the translated-content repo
 
-Over in the translated-content repo, decide what change you want to make, and
-then:
+Over in the translated-content repo, decide what change you want to make, and then:
 
 1. Create a new branch to make your changes in.
 
-2. Switch to your new branch and make the changes you want to make. You can keep
-   going back to `localhost:5042/<your_locale>` (e.g. `localhost:5042/fr` for
-   French) to test your changes and make sure the content looks how you want it
-   to look.
+2. Switch to your new branch and make the changes you want to make. You can keep going back to `localhost:5042/<your_locale>` (e.g. `localhost:5042/fr` for French) to test your changes and make sure the content looks how you want it to look.
 
-3. When you are satisfied with your changes, create a pull request and one of
-   our review teams will review it.
+3. When you are satisfied with your changes, create a pull request and one of our review teams will review it.
 
-4. Once the pull request has been merged, the edition may take up to 48 hours
-   (daily build and CDN caches). To see if your change has been deployed,
-   you can check on
-   [What's Deployed](https://whatsdeployed.io/s/16d/mdn/translated-content).
+4. Once the pull request has been merged, the edition may take up to 48 hours (daily build and CDN caches). To see if your change has been deployed, you can check on [What's Deployed](https://whatsdeployed.io/s/16d/mdn/translated-content).
```

Comment:
I think we removed references to this site in the other contribution docs. I would get rid of the last bit and align it [without stating anything about CDN / build](https://github.com/mdn/content/blob/main/CONTRIBUTING.md#pull-request-etiquette):

```suggestion
4. When your pull request is merged by a reviewer, the changes may take up to 48 hours to be visible.
```